templates/broadcom_icos_show_mac-address-table.textfsm

```diff
@@ -0,0 +1,14 @@
+Value VLAN (\S+)
+Value MAC (\S+)
+Value INTERFACE (\S+)
+Value INDEX (\S+)
+Value STATUS (\S+)
+
+Start
+  # Captures show mac-address-table for:
+  # Accton AS4610-54P, Accton AS5610-52X, Quanta LY2R, Quanta LB9, DNI AG3448P-R   
+  # Raw data is the same in the case of all those devices
+  ^\s*VLAN\s+ID\s+MAC\s+Address\s+Interface\s+IfIndex\s+Status
+  ^-+  -+  -+  -+

```

Comment:
```suggestion
  ^-+
```
I would remove the extra ones as they mark the headers as you have already made sure they exist.

Do you mind also adding the following right above the error line:
```
  ^\s*$$
```

Reply:
Corrected as requested